integration-tests/features/search/search.feature

```diff
@@ -87,3 +87,19 @@ Feature: Search
         When method get
         Then status 400
         And match $.Message == 'Not a valid language code.'
+
+
+    Scenario: cgov word boundries  #63
```

Comment:
Please update the scenario name to describe what you're testing.  Don't make the reader go looking elsewhere to find out what's being tested.  (i.e. Issue numbers are not good comments.)